exercises/practice/ledger/tests/ledger.cairo

```diff
@@ -0,0 +1,210 @@
+use ledger::{format_entries, Currency, Locale, Entry};
+
+#[test]
+fn empty_ledger() {
+    let currency = Currency::USD;
+    let locale = Locale::en_US;
+    let entries = array![];
+
+    let expected: Array<ByteArray> = array![
+        "Date       | Description               | Change       ",
+    ];
+    let result = format_entries(currency, locale, entries);
+
+    assert_eq!(result, expected);
+}
+
+#[test]
+#[ignore]
+fn one_entry() {
+    let currency = Currency::USD;
+    let locale = Locale::en_US;
+    let entries = array![
+        Entry { date: "2015-01-01", description: "Buy present", amount_in_cents: -1000 },
+    ];
+
+    let expected: Array<ByteArray> = array![
+        "Date       | Description               | Change       ",
+        "01/01/2015 | Buy present               |      ($10.00)",
+    ];
+    let result = format_entries(currency, locale, entries);
+
+    assert_eq!(result, expected);
+}
+
+#[test]
+#[ignore]
+fn credit_and_debit() {
+    let currency = Currency::USD;
+    let locale = Locale::en_US;
+    let entries = array![
+        Entry { date: "2015-01-01", description: "Buy present", amount_in_cents: -1000 },
+        Entry { date: "2015-01-02", description: "Get present", amount_in_cents: 1000 },
```

Comment:
Double check that all test entries are in correct order

[These two are not](https://github.com/exercism/problem-specifications/blob/ae6a54b3a195a012b4d24e86d934ba2181e2842b/exercises/ledger/canonical-data.json#L43-L52)